artifacts.html

```diff
@@ -0,0 +1,93 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Artifacts Page</title>
+</head>
+<body>
+<!-- --------------------------------HEADER CONTENT--------------------------------------------------------------------------------- -->
+<header>
+<!-- Social Media Images -->
+
+<a href="https://www.facebook.com">
+  <img class="socials" src="images/facebook.png" alt="image of the facebook icon to link to facebook" style="float: right; width: 1.5%; height: 1.5%; margin-right: 1%">
+</a>
+<a href="https://www.instagram.com">
+ <img class="socials" src="images/instagram.png" alt="image of the instagram icon to link to instagram" style="float: right; width: 1.5%; height: 1.5%; margin-right: 1%">
+</a>
+<a href="https://www.linkedin.com">
+ <img class="socials" src="images/linkedin.png" alt="iamge of the linkedin icon to link to linkedin" style="float: right; width: 1.5%; height: 1.5%; margin-right: 1%">
+</a>
+
+
+<!-- Social Media Icons - Source and Citation -->
+<!-- Citation for facebook icon -->
+<div>
+ Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Facebook">Facebook</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a>
+</div>
+<!-- Citation for instagram icon -->
+<div>
+ Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Instagram">Instagram</a> from <a href="https://www.flaticon.com/"     title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"     title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a>
+</div>
+<!-- Citation for linkedin icon -->
+<div>
+ Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Linkedin">Linkedin</a> from <a href="https://www.flaticon.com/"     title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"     title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a>
+</div>
+
+<br/>
+<br/>
+
+<!-- Banner for website with name and degree -->
+<h1 style="text-align: center"><u>KATRINA PROHASZKA</u></h1>
+<h3 style="text-align: center">MLIS in Information Management</h3>
+<div class="nav">
+
+
+<!-- Navigation Bar Menu -->
+ <div style="text-align: center; background-color: lightblue; width: 100%" >
+   <ul style="display:inline-block">
+     <li><a href="index.html">HOME </a></li>
+     <li>REFLECTION</li>
+     <li><a href="artifacts.html">ARTIFACTS</a></li>
+     <li><a href="contact.html">CONTACT</a></li>
+   </ul>
+ </div>
+</div> <!-- end nav div -->
+</header> <!-- --------------------------------END HEADER CONTENT--------------------------------------------------------------------------------- -->
+
+<!-- --------------------------------MAIN CONTENT--------------------------------------------------------------------------------- -->
+<main>
+<br/>
```

Comment:
You can remove the line breaks and use CSS in the next assignment to control your spacing/visual presentation